sdx-encrypted-submitter.go

```diff
@@ -9,10 +10,10 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"sdx-encrypted-submitter/authentication"
```

Comment:
this package import should be "github.com/ONSDigital/sdx-encrypted-submitter/authentication"

Reply:
Its part of the same package , and not in github yet so the import will fail anyway ?

Reply:
its to do with the fact that the code is hosted in github and making your code "go get"-able, please read the "Choose a good import path (make your package "go get"-able)" section in the go documentation https://blog.golang.org/organizing-go-code